src/app/home/client-trends-bar/client-trends-bar.component.ts

```
@@ -67,179 +60,135 @@ export class ClientTrendsBarComponent implements OnInit {
       case 'Day':
         while (labelsArray.length < 12) {
           date.setDate(date.getDate() - 1);
-          const transformedDate = this.dateUtils.formatDate(date, 'd/M');
-          labelsArray.push(transformedDate);
+          labelsArray.push(this.dateUtils.formatDate(date, 'd/M'));
         }
         break;
+
       case 'Week':
-        /** 1st January of present year */
         const onejan = new Date(date.getFullYear(), 0, 1);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)